components/alert/alert.css

```diff
@@ -0,0 +1,50 @@
+.alert-container {
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: space-around;
+    gap: 1rem;
+    background-color: var(--grey);
+    padding: 1rem;
+    border-radius: 0.5rem;
+    margin-top: 1rem;
+}
+
+.alert {
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: space-between;
+    align-items: center;
+    padding: 0.5rem 1rem 0.5rem 1rem;
```

Comment:
Alternative: "padding 0.5rem 1rem". it wil affect the same.

Reply:
okay Gopal i will do that .. Thankyou